src/test/resources/json/consignments_query_alldata.json

```diff
@@ -0,0 +1,3 @@
+{
+  "query": "{ consignments(limit: 2, currentCursor: \"consignment-ref1\") { pageInfo { startCursor, endCursor, hasNextPage, hasPreviousPage }, edges { node { seriesid, consignmentid, userid, totalFiles, fileChecks { antivirusProgress { filesProcessed }, checksumProgress { filesProcessed }, ffidProgress { filesProcessed }}, parentFolder, series { code }, transferringBody { name, tdrCode }, consignmentReference, createdDatetime, transferInitiatedDatetime, exportDatetime files { fileId metadata { clientSideFileSize, clientSideLastModifiedDate, clientSideOriginalFilePath, foiExemptionCode, heldBy, language, legalStatus, rightsCopyright } ffidMetadata { software softwareVersion binarySignatureFileVersion containerSignatureFileVersion method matches { extension identificationBasis puid } datetime }} currentStatus { upload }}}}}"
```

Comment:
I know a lot of our existing queries are like this, but is it valid to split the GraphQL query across multiple lines? In the past, I've found it quite hard to debug queries like this because you have to do so much horizontal scrolling.

Reply:
I'm not sure how straight forward this is, as json doesn't support multi-line strings.